tests/run-pass/fs.rs

```diff
@@ -40,6 +40,12 @@ fn main() {
     file.read_to_end(&mut contents).unwrap();
     assert_eq!(bytes, contents.as_slice());
 
+    // Test that seeking to the beginning and reading until EOF gets the text again.
```

Comment:
This explores *one* of the three variants of seek, I presume `SEET_SET`. Can you make sure all 3 are actually tested?